### PR TITLE
Updated configurations for k8s deployment

### PR DESCRIPTION
**Pull request by slowr**
### Description of PR

Updated configuration values on the kubernetes deployment.

What component(s) does this PR affect?

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [x] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: backend/core/configuration.py [hunk 1]
```diff
@@ -628,37 +619,56 @@ def check(self, data: Text) -> Dict:
...
+            is misdefined.
+            """
+            for section in data:
+                if section not in self.sections:
+                    raise ArtemisError("invalid-section", section)
+
+            data["prefixes"] = {k: flatten(v) for k, v in data["prefixes"].items()}
+            data["asns"] = {k: flatten(v) for k, v in data["asns"].items()}
+
+            Configuration.Worker.__check_prefixes(data["prefixes"])
```
Comment (vkotronis) [inline backend/core/configuration.py]: why not self here?

File: backend/core/configuration.py [hunk 2]
```diff
@@ -628,37 +619,56 @@ def check(self, data: Text) -> Dict:
...
+            self.__check_rules(data["rules"])
+            self.__check_monitors(data.get("monitors", {}))
+            Configuration.Worker.__check_asns(data["asns"])
```
Comment (vkotronis) [inline backend/core/configuration.py]: why not self here?

Comment (slowr) [reply backend/core/configuration.py]: these two don't need to access self context

File: backend/core/configuration.py (hunk 1 above)
Comment (slowr) [reply backend/core/configuration.py]: these two don't need to access self context

**Pull request merged**